lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -79,14 +79,7 @@ def signal_access_token_required
       response.set_header(ACCESS_TOKEN_REQUIRED_HEADER, "true")
     end
 
-    def jwt_expire_at
-      expire_at = request.env["jwt.expire_at"]
-      return unless expire_at
-
-      expire_at - 5.seconds # 5s gap to start fetching new token in advance
-    end
-
-    def add_top_level_redirection_headers(url: nil, ignore_response_code: false)
+    def add_top_level_redirection_headers(ignore_response_code: false)
```

Comment:
`url` argument is not used anywhere anymore, removing this. 
Extracted helper `RedirectForEmbedded.add_app_bridge_redirect_url_header(url, response)` to be reused elsewhere

Reply:
hmmm yea, although this method added more checks around _when_ to add the headers. I think I'll just add the existing behaviour back to avoid breaking behaviour for apps